package.json

```diff
@@ -16,6 +16,25 @@
     "node": "5.10.x"
   },
   "dependencies": {
+    "bootstrap-sass": "^3.3.7",
```

Comment:
We're using `npm shrinkwrap` which freezes all dependencies. All upgrades are manual.